src/styles/Country.css

```diff
@@ -0,0 +1,60 @@
+.top {
+  font-family: Lato, serif;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 0.1em;
+  background-color: #fc5192;
+  color: #fff;
+  font-size: 48px;
+  min-height: 250px;
+}
+
+.top div {
+  display: flex;
+  align-items: center;
+  gap: 20px;
+  padding: 20px;
+}
+
+.country-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.example-img {
+  width: 50%;
+}
+
+.country-container > * {
+  /* background-color: #fc5192; */
```

Comment:
- [ ] As mentioned above please remove the unused commented code 😉

Reply:
Thank you for your feedback. Removed.